### PR TITLE
Added:Remove a book by bookId

### PR DESCRIPTION
**Pull request by satya1-2**
first  I make a endpoint of remove and the write the business logic in Service class and then implement in service Interfcae

### CODE REVIEW

File: src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java [hunk 1]
```diff
@@ -13,4 +13,10 @@ public class Book {
     int id;
     String title, author, genre, isbn;
     Date publishedDate;
+
+
+    public boolean getId() {
```
Comment (ajinkyabridgelabz) [inline src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java:18]: Extremely wrong way to create a getter and setter

File: src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java [hunk 2]
```diff
@@ -13,4 +13,10 @@ public class Book {
     int id;
```
Comment (ajinkyabridgelabz) [inline src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java:13]: Modify it to have long, error created by @Mayurp375

**Review by ajinkyabridgelabz — CHANGES_REQUESTED** [review]

File: src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java (hunk 1 above)
Comment (satya1-2) [reply src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java:18]: resolved sir

File: src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java (hunk 2 above)
Comment (Mayurp375) [reply src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java:13]: resolved sir

**Comment by Mayurp375** [comment]
resolved the issue

**Comment by Mayurp375** [comment]
resolved the issue

**Pull request closed without merging**